### ISSUE TITLE
value-keyword-case doesn't ingore counter-reset or counter(name)

### ISSUE THREAD

**Issue by edg2s** [stylelint/stylelint#2406]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug

> Which rule, if any, is this issue related to?

value-keyword-case

> What CSS is needed to reproduce this issue?

e.g.

```css
.myClass {
	counter-reset: myCounter;
	content: '[' counter( myCounter ) ']';
	counter-increment: myCounter;
}

```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

7.8.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Node API (grunt-stylelint)

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

All three rules should pass

> What actually happened (e.g. what warnings or errors you are getting)?

Rules one and two give the following error;

```shell
Expected "myCounter" to be "mycounter"   value-keyword-case
```

The `counter-increment` rule is correctly ignored, and there are tests for it.

**Comment by edg2s** [comment]
See also #2404

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/issues/2407
